test/hotspot/jtreg/vmTestbase/gc/lock/LockerTest.java

```diff
@@ -33,41 +34,37 @@
  *
  * A number of threads is started. Each one locks and eats memory.
  */
-public class LockerTest extends ThreadedGCTest implements GarbageProducerAware, GarbageProducer1Aware, LockersAware {
+public class LockerTest extends ThreadedGCTest implements LockersAware {
 
-    private GarbageProducer garbageProducer;
-    private GarbageProducer garbageProducer1;
     private Lockers lockers;
     private long objectSize = 1000;
 
     private class Worker implements Runnable {
 
-        byte[] rezerve = new byte[1024 * 1024];
-        private Locker locker = lockers.createLocker(garbageProducer1.create(objectSize));
+        byte[] rezerve = new byte[1024];
+        private Locker locker = lockers.createLocker(rezerve);
 
         public Worker() {
             locker.enable();
         }
 
         public void run() {
-            locker.lock();
-            GarbageUtils.eatMemory(getExecutionController(), garbageProducer);
-            locker.unlock();
+            ExecutionController stresser = getExecutionController();
+            // Use only 30% of the heap.
+            final long testMemory = 3 * Runtime.getRuntime().maxMemory() / 10;
```

Comment:
might be better to rename to testMemorySize

Reply:
I prefer to leave it as engageGC, it shows the intent better.

Reply:
I meant rename testMemory to testMemorySize

Reply:
Oh! I misunderstood. Will submit changes.